XRTK.SDK/Packages/com.xrtk.sdk/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -1121,111 +1113,34 @@ private void SetHiddenHandles()
 
         private static void GetCornerPositionsFromBounds(Bounds bounds, ref Vector3[] positions)
         {
-            int numCorners = 1 << 3;
+            const int numCorners = 1 << 3;
```

Comment:
Should probably make this a class level constant to stop recreating it

Reply:
Sure 👍